kolibri/plugins/coach/class_summary_api.py

```diff
@@ -37,6 +36,10 @@
 COMPLETED = "Completed"
 
 
+exam_viewset = ExamViewset()
+lesson_viewset = LessonViewset()
```

Comment:
Could you add a comment here explaining what is happening here? Like -- why are we instantiating them at the module level? Does this only matter for the life of a single request or are the instances used across multiple requests to some benefit?